core/client/app/components/gh-user-invited.js

```diff
@@ -16,8 +17,10 @@ export default Ember.Component.extend({
     actions: {
         resend: function () {
             var user = this.get('user'),
-                notifications = this.get('notifications');
+                notifications = this.get('notifications'),
+                self = this;
```

Comment:
Not sure if this was discussed somewhere, but wouldn't it make sense to switch to arrow-functions here to make lines like the above obsolete?